### ISSUE TITLE
Seems like an upgrade to 15.8 has broken printing ...

### ISSUE THREAD

**Issue by aiamuzz** [linuxdeepin/developer-center#665]
Hi,

I have recently updated to 15.8 ... seems like i am unable to print anything ... i tried adding the printer anew ... i tried printing from the command line using 'lpr' ... everything seems to just queue up ... with nothing printing out of the printer.

**Comment by hualet** [comment]
what type is it?  remotely or locally?

**Comment by aiamuzz** [comment]
@hualet  ... i have a network printer(IP) and its not printing ... all jobs are getting queued up but nothing is making it to the printer !!!

**Comment by aiamuzz** [comment]
ok ... here is additional behavior ...

My daily driver Deepin OS install upgraded from 15.7 ... demonstrated this printer issue ... i am facing an issue with after the first update having moved to 15.8 ... there was a small(less than 20MB) update ... after installing that i am not able to boot into the machine ... it get stuck at a point at boot ...

so fed up of this issue ... last night I have installed a fresh 15.8 out of an ISO ... the printer his working fine in this fresh install.

Seems like this printing issue is encountered on a machine upgraded from 15.7 ...

**Comment by hualet** [comment]
OK, which point did it stuck at BTW, maybe we can help you with it.

**Comment by aiamuzz** [comment]
> OK, which point did it stuck at BTW, maybe we can help you with it.

IF YOU GUYS DON'T HELP ME FIX THIS ISSUE IN SPITE OF HAVING DEEPIN OS INSTALLED ON 3 PARTITIONS ... THEN YOU ARE BASICALLY TELL ME TO 'FUCK-OFF' ... YOU HAVE NO IDEA HOW FRUSTRATING ITS BEEN ... WHENEVER I END UP WITH THIS ISSUE ... I KEEP TRYING TO BOOT AT TIMES I HAVE HAD TO TRY 10 TIMES TO LAND 1 SUCCESSFUL BOOT !!!

PLEASE LOOK INTO THIS ISSUE ... [HERE](https://github.com/linuxdeepin/developer-center/issues/440#issuecomment-439800977)

**Comment by aiamuzz** [comment]
**NOW EVEN MY FRESH INSTALL OF DEEPIN 15.8 LAST NIGHT IS NOT BOOTING AND ITS GETTING STUCK ...** 

if a fresh install can't resolve this issue ... then i have no way out !!!

Really appreciated help from your team to identify and resolve this issue !!!

**Comment by aiamuzz** [comment]
@hualet ...

**Just to confirm ... the printing issue continues on the upgraded installation.**
Its working fine on a fresh install though

**Comment by hualet** [comment]
@aiamuzz  I'm not very familiar with printing stack on linux... what if you reconfigure the printer ?

**Comment by aiamuzz** [comment]
> @aiamuzz I'm not very familiar with printing stack on linux... what if you reconfigure the printer ?

I have already tried that ... and only after that i have opened the issue !!!

**Comment by justforlxz** [comment]
Sorry, this issue will be closed soon. If it is necessary to discuss it again, please create a new issue.
